test/cmd/deployments.sh

```diff
@@ -132,6 +132,14 @@ os::cmd::expect_success 'oc delete hpa/test-deployment-config'
 echo "autoscale: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/deployments/setimage"
+os::cmd::expect_success 'oc create -f test/integration/testdata/test-deployment-config.yaml'
+os::cmd::expect_success 'oc set image dc/test-deployment-config ruby-helloworld=myshinynewimage'
+os::cmd::expect_success_and_text "oc get dc/test-deployment-config -o jsonpath='{.spec.template.spec.containers[0].image}'" "myshinynewimage"
+os::cmd::expect_success 'oc delete dc/test-deployment-config'
```

Comment:
Actually, this isn't a cluster resource -- do we need to explicitly delete it? Cluster resources we want to delete in cleanup blocks (ex in [`admin.sh`](https://github.com/openshift/origin/blob/master/test/cmd/admin.sh#L5-L21)) as that's necessary to make the scripts re-entrant. Namespaced resources can be left to get reaped by the namespace deletion.